core/trino-main/src/main/java/io/trino/metadata/Metadata.java

```diff
@@ -137,6 +137,12 @@
      */
     TableStatistics getTableStatistics(Session session, TableHandle tableHandle, Constraint constraint);
 
+    /**
+     * Returns {@code true} when {@link #getTableStatistics} returns more useful (e.g. more accurate) result
+     * for {@code oldTableHandle} than for {@code newTableHandle}.
```

Comment:
nit: the docstring describes a single `oldTableHandle`, but the method uses a list (`oldTableHandles `).